FLT/DedekindDomain/FiniteAdeleRing/BaseChange.lean

```diff
@@ -63,8 +63,6 @@ def comap (w : HeightOneSpectrum B) : HeightOneSpectrum A where
   isPrime := Ideal.comap_isPrime (algebraMap A B) w.asIdeal
   ne_bot := mt Ideal.eq_bot_of_comap_eq_bot w.ne_bot
 
-open scoped algebraMap
```

Comment:
I feel like this only causes trouble (e.g. the `show (r : adicCompletion K (comap A w)) = @UniformSpace.Completion.coe' K this r from rfl` below) and we don't really use it anyway.